src/main/java/subway/line/Line.java

```diff
@@ -0,0 +1,71 @@
+package subway.line;
+
+import subway.station.StationResponse;
+
+import java.util.List;
+
+public class Line {
+    private Long id;
+    private String name;
+    private String color;
+    private Long startStationId;
+    private Long endStationId;
+
+    public Line() {
+    }
+
+    public Line(Long id, String name, String color, Long startStationId, Long endStationId) {
+        this.id = id;
+        this.name = name;
+        this.color = color;
+        this.startStationId = startStationId;
+        this.endStationId = endStationId;
+    }
+
+    public Line(String name, String color, Long startStationId, Long endStationId) {
+        this.name = name;
+        this.color = color;
+        this.startStationId = startStationId;
+        this.endStationId = endStationId;
```

Comment:
같은 로직 뿐이라면, _더 많은 일을_ 하는 쪽으로 delegate 하면 좋을것 같아요
```suggestion
        this(null, name, color, startStationId, endStationId);
```